Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/VanillaMemeEdit.java

```diff
@@ -22,6 +38,40 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    public void onShareClick(View v){
```

Comment:
I think on lines 93-98 you accidentally erased my method for the gallery button...or was it moved somewhere else too?